flow/alerting/classifier.go

```diff
@@ -249,10 +249,38 @@ func GetErrorClass(ctx context.Context, err error) (ErrorClass, ErrorInfo) {
 
 	var myErr *mysql.MyError
 	if errors.As(err, &myErr) {
-		return ErrorOther, ErrorInfo{
+		// https://mariadb.com/kb/en/mariadb-error-code-reference
+		myErrorInfo := ErrorInfo{
 			Source: ErrorSourceMySQL,
 			Code:   strconv.Itoa(int(myErr.Code)),
 		}
+		switch myErr.Code {
+		case 1037, 1038, 1041, 3015:
+			return ErrorNotifyOOM, myErrorInfo
```

Comment:
Do you mean oom source?
```suggestion
			return ErrorNotifyOOMSource, myErrorInfo
```